tests/cases/fourslash/importNameCodeFix_jsExtension.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path="fourslash.ts" />
+
+// @moduleResolution: node
+// @noLib: true
+
+// @Filename: /a.ts
+////export function a() {}
+
+// @Filename: /b.ts
+////export function b() {}
+
+// @Filename: /c.ts
+////import { a } from "./a.js";
```

Comment:
For documentation purposes, it might be nice to add a test where some imports have extensions and others do not.  (It looks like absolute paths might also be treated specially.)